esmf_regrid/schemes.py

```diff
@@ -225,33 +225,35 @@ def _regrid_rectilinear_to_rectilinear__perform(src_cube, regrid_info, mdtol):
 
 def regrid_rectilinear_to_rectilinear(src_cube, grid_cube, mdtol=0):
     """
-    Regrid rectilinear cube onto another rectilinear grid.
+    Regrid rectilinear :class:`~iris.cube.Cube` onto another rectilinear grid.
 
-    Return a new cube with data values calculated using the area weighted
-    mean of data values from the cube src_cube regridded onto the
-    horizontal grid of grid_cube.
+    Return a new :class:`~iris.cube.Cube` with :attr:`~iris.cube.Cube.data`
+    values calculated using the area weighted
+    mean of :attr:`~iris.cube.Cube.data` values from ``src_cube`` regridded onto the
+    horizontal grid of ``grid_cube``.
 
     Parameters
     ----------
-    src_cube : cube
-        An urectilinear instance of iris.cube.Cube that supplies the data,
+    src_cube : :class:`iris.cube.Cube`
+        An un-rectilinear instance of :class:`~iris.cube.Cube` that supplies the data,
```

Comment:
I believe this should have been "rectilinear". With that said, I have a feeling we also support curvilinear grids now. This could be updated in a later PR if needs be since "rectilinear" is in the name of the function and I can't think up a good name to replace it that covers both rectilinear and curvilinear grids.

Reply:
4a82f52691364fb89498abaa42867a46ca0e63cf